src/libinjection_error.h

```diff
@@ -0,0 +1,26 @@
+/**
+ * LibInjection Project
+ * BSD License -- see `COPYING.txt` for details
+ *
+ * https://github.com/libinjection/libinjection/
+ *
+ */
+
+#ifndef LIBINJECTION_ERROR_H
+#define LIBINJECTION_ERROR_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef enum injection_result_t {
+    RESULT_FALSE = 0,
+    RESULT_TRUE = 1,
+    RESULT_ERROR = -1
```

Comment:
```suggestion
    LIBINJ_RESULT_FALSE = 0,
    LIBINJ_RESULT_TRUE = 1,
    LIBINJ_RESULT_ERROR = -1
```
I recommend using the `LIBINJ` prefix (or some other clarifying prefix) here instead of these generic forms. It should be noted that the embedding application may use other libraries with similar enumeration types.

Reply:
Went for full `LIBINJECTION_` prefix.